src/app/lib/config/services/persian.js

```diff
@@ -69,7 +69,7 @@ export const service = {
       brandTitle: 'رادیو فارسی بی‌بی‌سی',
       brandDescription: 'پادکست چشم‌انداز بامدادی رادیو بی‌بی‌سی',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p02h1lnx.jpg',
+        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p0bq9rkk.jpg',
         alt: 'رادیو فارسی بی‌بی‌سی',
```

Comment:
Is the alt text still the same?

Reply:
hi, I've double checked with the Persian team. yes, having the same alt tag is correct.